DuckDuckGo/HomePage/Model/HomePageSettings/HomePageSettingsModel.swift

```diff
@@ -23,12 +23,37 @@ import PixelKit
 import SwiftUI
 import SwiftUIExtensions
 
+protocol SettingsVisibilityModelPersistor {
+    var didShowSettingsOnboarding: Bool { get set }
+}
+
+final class UserDefaultsSettingsVisibilityModelPersistor: SettingsVisibilityModelPersistor {
+    @UserDefaultsWrapper(key: .homePageDidShowSettingsOnboarding, defaultValue: false)
+    var didShowSettingsOnboarding: Bool
+}
+
 extension HomePage.Models {
     /**
-     * This tiny model is used by HomePageViewController to expose a setting to control settings visibility
+     * This tiny model is used by HomePageViewController to expose a setting to control settings visibility,
+     * as well as to keep track of the settings onboarding popover.
      */
     final class SettingsVisibilityModel: ObservableObject {
         @Published var isSettingsVisible: Bool = false
+
+        var didShowSettingsOnboarding: Bool {
```

Comment:
🔍 Should this be called `didShowSettingsOnboardingPopover`? Reading it seems like a setting to know if the settings/customize screen was opened.

Reply:
I decided to skip "popover" in the name in case we have a different type of onboarding in the feature. Other than that the name contains "Onboarding" so I thought it would be clear that it's about the onboarding :) happy to change it anyway.